tools/reviewer/web/README.md

```diff
@@ -2,7 +2,8 @@
 Run `ng install` for the first time.
 
 Run `npm run protoc` to generate proto functions.
-You need installed google-protobuf proto3 on your computer.
+You need installed google-protobuf version 3.5 or above on your computer.
```

Comment:
Let's rephrase:
You need google-protobuf 3.5 or above installed on your machine.
Supported on Linux and Mac only.